calculator_v5.rb

```diff
@@ -0,0 +1,93 @@
+    # adds and assigns result to variable
+    result = num1 + num2
+    # outputs formula and result celebration.
+    puts "#{num1} + #{num2} = #{result}"
+    puts "The answer is #{result}! Woohoo!!! Math is so cool."
+
+  # if operation is equal to values indicating subtraction
+  elsif operation == "-" || operation == "subtract" || operation == "minus" || operation == "subtraction" || operation == "subtracting"
+    # subtracts and assigns result to variable
+    result = num1 - num2
+    # outputs formula and result celebration.
+    puts "#{num1} - #{num2} = #{result}"
+    puts "The answer is #{result}! Woohoo!!!"
+
+  # if operation is equal to values indicating multiplication
+  elsif operation == "*" || operation == "x" || operation == "multiply" || operation == "multiplication" || operation == "multiplying"
+    # multiples and assigns result to variable
+    result = num1 * num2
+    # outputs formula and result celebration.
+    puts "#{num1} * #{num2} = #{result}"
+    puts "The answer is #{result}! Woohoo!!!"
+
+  # if operation is equal to values indicating division
+  elsif operation == "/" || operation == "divide" || operation == "division" || operation == "dividing"
+    # outputs error if attempt to divide by 0. Otherwise, compute result.
+    if num2 == 0
+      puts "That's ridiculous! You can't divide by zero! Goodbye now!"
+    else
+      # divides and assigns result to variable
+      result = num1 / num2
+      # outputs formula and result celebration.
+      puts "#{num1} / #{num2} = #{result}"
+      puts "The answer is #{result}! Woohoo!!!"
+    end
+
+  # if operation is equal to values indicating exponent
+  elsif operation == "^" || operation == "**" || operation == "exponents" || operation  == "exponent" || operation == "power"
+    # takes num1 to the power of num2 and assigns result to variable
+    result = num1**(num2)
+    # outputs formula and result celebration.
+    puts "#{num1}^#{num2} = #{result}"
+    puts "The answer is #{result}! Woohoo!!!"
+
+  # if operation is equal to values indicating modulo
+  elsif operation == "%" || operation == "modulus" || operation == "modulo" || operation == "remainder"
+  # calculates remainder of num1 divided by num2 and assigns result to variable
+    result  = num1 % num2
+    # outputs formula and result celebration.
+    puts "#{num1} % #{num2} = #{result}"
+    puts "The answer is #{result}! Woohoo!!!"
+
+  # if other operation value, outputs error
+  else
+    puts "I'm sorry, I don't know how to do #{operation}. Goodbye!"
+  end
+end
+
+# Outputs query for first number, then assigns user entered value to num1
+print "What is the first number? "
+num1 = gets.chomp.delete(",")
+
+# Outputs query for second number, then assigns user entered value to num2
+print "Fantastic! How about the second number? "
+num2 = gets.chomp.delete(",")
+
+numbers = [ "0", "1", "2", "3", "4", "5", "6", "7", "8", "9" ]
```

Comment:
This is just trivia because I think Ruby is neat. Here's three ways to write the same line:

```ruby
numbers = [ "0", "1", "2", "3", "4", "5", "6", "7", "8", "9" ]
numbers = ("0".."9").to_a
numbers = %w(0 1 2 3 4 5 6 7 8 9)
```

Yay Ruby!